### ISSUE TITLE
Make it possible to disable the Docker health check service of Butler CW

### ISSUE THREAD

**Issue by mountaindude** [ptarmiganlabs/butler-cw#35]
Add config option and corresponding code in Butler CW:

```
  # Docker health checks are used when running Butler CW as a Docker container. 
  # The Docker engine will call the container's health check REST endpoint with a set interval to determine
  # whether the container is alive/well or not.
  # If you are not running Butler CW in Docker you can safely disable this feature. 
  dockerHealthCheck:
    enabled: true         # Control whether a REST endpoint will be set up to serve Docker health check messages
    port: 12398           # Port the Docker health check service runs on (if enabled)

```

**Comment by mountaindude** [comment]
Implemented by a029141

**Comment by mountaindude** [comment]
Released in [version 2.3.0](https://github.com/ptarmiganlabs/butler-cw/releases/tag/v2.3.0).
